nistats/tests/test_dmtx.py

```diff
@@ -43,6 +43,7 @@ def design_matrix_light(
                                           drift_model, period_cut, drift_order, fir_delays,
                                           add_regs, add_reg_names, min_onset)
     _, matrix, names = check_design_matrix(dmtx)
+    print matrix, names
```

Comment:
line 46: print will crash in python3, no?

Reply:
Also, we should not print in tests.